src/database/mod.rs

```diff
@@ -121,113 +125,105 @@ impl Database {
         Ok(Self { pool })
     }
 
+    pub async fn is_empty(&self) -> Result<bool, Error> {
```

Comment:
Should be called `has_no_identities` or something similar. Currently calling `database.is_empty()` makes sense, but if we add more tables in the future this name will be misleading (but will probably not be changed at that point because the future versions of us will lack this context).